index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
           </pre>
         </section>
       </section>
+
+      <section>
+        <h3>Data Schemas</h3>
+
+        <p>
+Data schemas are useful when enforcing a specific structure on a given
+collection of data. There are at least two types of data schemas that this
+specification considers:
+        </p>
+
+        <ul>
+          <li>
+Data verification schemas, which are used to <a>verify</a> that the structure
+and contents of a <a>credential</a> or <a>verifiable credential</a> conform to a
+published schema.
+          </li>
+          <li>
+Data encoding schemas, which are used to map the contents of a
+<a>verifiable credential</a> to an alternative representation format, such as a
+format used in a zero-knowledge proof.
```

Comment:
We have no work items that actually do this... I feel this paragraph is harmful.